embrace-gradle-plugin/src/main/java/io/embrace/android/gradle/plugin/EmbraceGradlePluginDelegate.kt

```diff
@@ -57,6 +57,11 @@ class EmbraceGradlePluginDelegate {
 
         taskRegistrar.registerTasks()
 
+        // Register React Native specific tasks when React Native plugin is also applied
+        project.pluginManager.withPlugin("com.facebook.react") {
```

Comment:
Is this available in all supported RN versions? I remember it used to be react.gradle

Reply:
yes, at least for the ones we support (0.71+)